rasa/core/tracker_store.py

```diff
@@ -965,7 +965,15 @@ def _create_database_and_update_engine(self, db: Text, engine_url: "URL") -> Non
 
         self._create_database(self.engine, db)
         self.engine.dispose()
-        engine_url.database = db
+        engine_url = URL(
```

Comment:
the URL is immutable due to https://docs.sqlalchemy.org/en/14/changelog/changelog_14.html#change-41f7a69f16c0de9908f5e37c68b6d6d0 . Didn't use the new setter as it's not available in 1.3

Reply:
and we need 1.3 compatibility as Rasa X is using that as far as I know.